test/bdd/fixtures/auth-rest/oidc-config/providers.yaml

```diff
@@ -9,5 +9,5 @@ providers:
     url: https://third.party.oidc.provider.example.com:5555/
     clientID: hub-auth
     clientSecret: hub-auth-secret
-    name: Universal Bank
-    logoURL: https://upload.wikimedia.org/wikipedia/en/a/a4/Universal_Bank.svg
+    name: Demo Sign-Up Partner
+    logoURL: https://media.githubusercontent.com/media/trustbloc/edge-agent/main/cmd/wallet-web/src/assets/img/demo_provider.svg
```

Comment:
We can add 2 more keys to support signUp and signIn texts.

```
providers:
  demoProvider:
    url: https://third.party.oidc.provider.example.com:5555/
    clientID: hub-auth
    clientSecret: hub-auth-secret
    name: Demo OIDC 
    signUpText: Demo Sign-Up Partner
    signInText: Demo Sign-In Partner
    logoURL: <urlLink>
```

We need a way to configure logos in static server to serve it (shouldn't point it to github files). One way is to mount logos to hub-auth docker image and serve the static content.

Reply:
PR was updated to fetch logos from static url in hub-auth - https://github.com/trustbloc/hub-auth/pull/130/files#diff-db24e648b0d856e6d588e7ea854837eb7e7bc460734d5b48752cabe7d7047279R33